File: src/main/java/de/litexo/model/external/ExportModel.java
package de.litexo.model.external;

import de.litexo.commands.PauseCommand;
import de.litexo.commands.UnpauseCommand;
import de.litexo.events.OpenttdTerminalSnapshotEvent;
import de.litexo.events.OpenttdTerminalUpdateEvent;
import lombok.Data;

@Data
public class ExportModel {
    private OpenttdTerminalUpdateEvent openttdTerminalUpdateEvent;
    private OpenttdTerminalSnapshotEvent openttdTerminalSnapshotEvent;
    private UnpauseCommand unpauseCommand;
    private PauseCommand pauseCommand;

    private ServiceError serviceError;
}

File: ui/src/app/api/models/export-model.ts
/* tslint:disable */
/* eslint-disable */
import { OpenttdTerminalSnapshotEvent } from './openttd-terminal-snapshot-event';
import { OpenttdTerminalUpdateEvent } from './openttd-terminal-update-event';
import { PauseCommand } from './pause-command';
import { ServiceError } from './service-error';
import { UnpauseCommand } from './unpause-command';
export interface ExportModel {
  openttdTerminalSnapshotEvent?: OpenttdTerminalSnapshotEvent;
  openttdTerminalUpdateEvent?: OpenttdTerminalUpdateEvent;
  pauseCommand?: PauseCommand;
  serviceError?: ServiceError;
  unpauseCommand?: UnpauseCommand;
}
